test/presubmit-tests.sh

```diff
@@ -21,9 +21,7 @@
 # Use the flags --build-tests, --unit-tests and --integration-tests
 # to run a specific set of tests.
 
-export GO111MODULE=on
-
-source $(dirname "${BASH_SOURCE[0]}")/../scripts/presubmit-tests.sh
+source $(dirname "$0")/../../vendor/knative.dev/presubmit-tests.sh
```

Comment:
too many `..`
Please use `$(dirname "${BASH_SOURCE[0]}")` instead of `$(dirname "$0")`